qiskit/ignis/experiments/base/generator.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, name: str, qubits: Union[int, List[int]]):
             self._qubits = list(range(self._num_qubits))
         else:
             self._qubits = list(qubits)
-            self._num_qubits = len(self._qubits)
+            self._num_qubits = max(self._qubits)+1
```

Comment:
why we need to change this? this impacts to other MVPs.

Reply:
Hmmm perhaps this is not the right place to change this. The use case is the following: run a cal on qubits 0 and 3 so you specify `qubits=[0, 3]`. Initially, I was creating the quantum circuit with `QuantumCircuit(self._num_qubits, self._num_qubits)` so with `self._num_qubits=len(qubits)` I could not do `qc.x(3)`. But perhaps I should change the creation of the circuit from `QuantumCircuit(self._num_qubits, self._num_qubits)` to `QuantumCircuit(max(self._qubits)+1, max(self._qubits)+1)`. What do you prefer?